tx-pool/src/component/pool_map.rs

```diff
@@ -119,7 +119,7 @@ impl PoolMap {
 
     #[cfg(test)]
     pub(crate) fn add_proposed(&mut self, entry: TxEntry) -> Result<bool, Reject> {
-        self.add_entry(entry, Status::Proposed)
+        self.add_entry(entry, Status::Proposed).map(|_| true)
```

Comment:
Any background for this change? Does it revert the "Direct Parent" patch?

Reply:
`add_entry` was changed. Anyway, I just fixed `add_entry` to return `(succ, evicts)`, so that we can still get a flag whether the entry was inserted successfully.